cirq/circuits/moment.py

```diff
@@ -14,9 +14,11 @@
 
 """A simplified time-slice of operations within a sequenced circuit."""
 
-from typing import Any, Iterable, TypeVar, Callable, Sequence
+from typing import Any, Iterable, TypeVar, Callable, Sequence, TYPE_CHECKING
 
-from cirq import ops
+if TYPE_CHECKING:
+    # pylint: disable=unused-import
+    from cirq import ops
```

Comment:
Why did you need to move this into a type-checking only block? There shouldn't be any dependence of ops on circuits, and if there is the cycle should be broken there.